common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -914,7 +914,7 @@ div.problem .action {
     }
   }
 
-  .save {
+  .check {
```

Comment:
Just move this change to the [theme](https://github.com/Edraak/edraak-2015-theme).